packages/errors/src/index.ts

```diff
@@ -42,7 +42,7 @@ export interface ErrorWithOptionalCode extends Error {
 }
 
 /**
- * Test to see if a given exception is an AWS Throttling Exception
+ * Test to see if a given exception is a Throttling Exception.
```

Comment:
Things like this took some time to suss out. In this case ThrottlingException is a Cumulus-defined error, not an aws-sdk error:
```
export class ThrottlingException extends Error 
```

Reply:
It makes me a little worried about missing stuff but there are a few instances like this where it's ambiguous where the error is actually coming from.